tools/update-table.js

```diff
@@ -0,0 +1,49 @@
+const bent = require('bent')
+const HEADER = `
+// THIS FILE IS GENERATED, DO NO EDIT MANUALLY
+// For more information see the README.md
+'use strict'
+
+const { numberToBuffer } = require('../src/util')
+
+const table = {
+`
+const FOOTER = `
+}
+for (const [k, v] of Object.entries(table)) {
+  table[k] = numberToBuffer(v)
+}
+module.exports = Object.freeze(table)
+`
+
+const get = bent('string')
+const url = 'https://raw.githubusercontent.com/multiformats/multicodec/master/table.csv'
+const fs = require('fs')
+
+const parse = async function * () {
+  let str = await get(url)
+  let lines = str.split('\n')
+  let header = lines.shift()
+  for (let line of lines) {
+    if (!line.length) continue
+    let [name, tag, code, desc] = line.split(',')
+    yield { name: name.trim(), tag: tag.trim(), code: parseInt(code.trim(), 16) }
+  }
+}
+
+const run = async () => {
+  let str = HEADER
+  let first = true
+  for await (let entry of parse()) {
+    key = entry.name
+    if (key.includes('-')) key = `'${key}'`
```

Comment:
I'm unsure if manually creating the `table` object makes sense, or if it would be better to just push everything into an object and put that into the file. Though if it works, why changing :) I leave that up to you, you have to maintain it :)